core/realm.rs

```diff
@@ -8,21 +8,43 @@ use anyhow::Error;
 use std::cell::RefCell;
 use std::collections::HashMap;
 use std::collections::HashSet;
+use std::hash::BuildHasherDefault;
+use std::hash::Hasher;
+use std::marker::PhantomData;
 use std::option::Option;
 use std::rc::Rc;
 use v8::HandleScope;
 use v8::Local;
 
+// Hasher used for `unrefed_ops`. Since these are rolling i32, there's no
+// need to actually hash them.
+#[derive(Default)]
+pub(crate) struct IdentityHasher(u64, PhantomData<i32>);
```

Comment:
Why is `PhantomData` needed here?

Reply:
I guess it's not needed. I'll remove it.